system/lib/wasm_worker/library_wasm_worker.c

```diff
@@ -105,15 +105,19 @@ void emscripten_lock_waitinf_acquire(emscripten_lock_t *lock) {
 }
 
 bool emscripten_lock_busyspin_wait_acquire(emscripten_lock_t *lock, double maxWaitMilliseconds) {
+  // Note: using emscripten_get_now() over emscripten_performance_now() to allow
+  // compatibility with Audio Worklets (but for workers this will defer to the
+  // performance call with its higher granularity). We keep the non-busy locks
+  // using the performance version since they have waits, incompatible with AWs.
```

Comment:
I'm not really sure we need this comment, and I think we we should probably just convert everything to `emscripten_get_now`.  

If you do go with a comment how about "Use emscripten_get_now over emscripten_performance_now since the former is available in all contexts and uses performance.now under the hood where available.".

Reply:
All the other lock types used waits, so it was a limited set of functions that could work. There’s no current benefit, unless I’m missing something, to changing all uses over to plain `now`.

Reply:
I thought it useful to leave a comment in case someone later thinks to change it.

It does need docs adding in the web audio page to explain what locks can be used. I’ll do this, and finish the tests, in the next days.

Reply:
I guess i just don't see the point of `emscripten_performance_now` at all.  I'm not sure why it was ever added.  Why wouldn't all callsites simply prefer `emscripten_get_now`?   The works in more places (i.e. all places) and lowers the same thing when `performance.now` is available.

Reply:
I've had time to think about this: `emscripten_performance_now()` when supported is very lightweight (it maps directly to the browser's call). `emscripten_get_now()` has a bit of indirection, and for high performance code I'd prefer the option to know I'm avoiding indirection.

For this PR I've kept the change to the parts that need changing to prove out a set of lock calls work.

Reply:
FWIW emscripten_get_now() maps directly to `performance.now()` whenever `performance.now() is available.  See https://github.com/emscripten-core/emscripten/blob/7342ee809ac4bcff958d61ac327530ed4b6eaf38/src/library.js#L1393-L1396.  There is no indirection.

Reply:
Good to know, thanks.